ONSdigital/florence
# Fix/pass collection id header with requests

### What

- Add collection cookie header setter
- Tidied access token header setter to use common consts

### How to test

Run both Zebedee and Dataset-API with `ENABLE_PERMISSIONS_AUTH=true` and try use the CMD journey in Florence. You should notice no differences and should be able to use them

### Who can review

Anyone
